SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -635,6 +635,12 @@ void USpatialReceiver::HandleActorAuthority(const Worker_ComponentSetAuthorityCh
 					Actor->Role = ROLE_Authority;
 					Actor->RemoteRole = ROLE_SimulatedProxy;
 
+					// bReplicates is not replicated, but this actor is replicated
+					if (!Actor->GetIsReplicated())
+					{
+						Actor->SetReplicates(true);
```

Comment:
This is set in the AuthorityChange as I need the server to be authoritative on the Actor to be able to `SetReplicates`. The server doesn't yet have authority in `TryGetOrCreateActor`